backend/rest/server.go

```diff
@@ -86,6 +87,7 @@ func NewAPIMux() *mux.Router {
 	registerApi(s)
 	registerFilters()
 
+	r.PathPrefix("/swagger-ui/").Handler(http.StripPrefix("/swagger-ui/", http.FileServer(http.Dir("../swagger-ui"))))
```

Comment:
According to the deployment environment, choose whether to enable the swagger service. Only dev, qa environment needs to be enabled.

Reply:
ok